.github/workflows/superset.yml

```diff
@@ -0,0 +1,205 @@
+name: Superset
+
+on:
+  # only build on direct push to `master` branch
+  push:
+    branches: [ master ]
+    paths:
+    - superset/**/*.py
+    - tests/**
+    - setup.py
+    - requirements*.txt
+  # but also build on pull requests to any branch
+  # (the so-called feature branch)
+  pull_request:
+    paths:
+    - superset/**/*.py
+    - tests/**
```

Comment:
Should this include `superset-frontend/**` ?

Reply:
This file is purely for python code. Frontend code checks are done in other workflows. Maybe I this rename this file to avoid confusion.